operator-pipeline-images/operatorcert/__init__.py

```diff
@@ -221,7 +221,7 @@ def get_files_added_in_pr(
     added_files_names = []
     modified_files = []
     for file in rsp.json().get("files", []):
-        if file["status"] == "added":
+        if file["status"] == "added" or file["filename"].endswith("ci.yaml"):
```

Comment:
I suppose this is ok but it's a little confusing since we want to allow `ci.yaml` to be _modified_. Would it be better to allow `ci.yaml` to be appended to `modified_files` and add an exception (with a clear log message) further down? I'm not sure of the implications of excluding it from the returned list in `added_files_names` though.